### PR TITLE
Sync Primus packaging with Debian

### PR DESCRIPTION
**Pull request by bluca**
Add dependency to xserver-xorg-video-intel or xsever-xorg-core for SNA for Wily and Xenial builds, and remove -dbg from Xenial build. Already tested on the testing PPA, looks fine.

### CODE REVIEW

File: wily/primus/debian/control [hunk 1]
```diff
@@ -3,7 +3,8 @@ Section: utils
 Priority: extra
 Maintainer: The Bumblebee Project <bumblebee@lists.launchpad.net>
 Uploaders: Bruno Pagani <bruno.n.pagani@gmail.com>,
- Dmitry Zhurikhin <zhurikhin@gmail.com>
+ Dmitry Zhurikhin <zhurikhin@gmail.com>,
+ Luca Boccassi <luca.boccassi@gmail.com>,
```
Comment (Lekensteyn) [inline wily/primus/debian/control]: Are trailing commas OK?

File: xenial/primus/debian/rules [hunk 2]
```diff
@@ -46,4 +46,4 @@ override_dh_install:
 	dh_install
 
 override_dh_strip:
-	dh_strip --dbg-package=primus-libs-dbg
+	dh_strip --dbgsym-migration='primus-libs-dbg (<< 20150328-1~xenialppa2)'
```
Comment (Lekensteyn) [inline xenial/primus/debian/rules]: The next version is 20150328-2~xenialppa1, is this number here intentional?

Comment (bluca) [inline xenial/primus/debian/rules]: Nope, nice catch (works anyway but it is incorrect)

File: wily/primus/debian/control (hunk 1 above)
Comment (bluca) [inline wily/primus/debian/control]: Yep, I like them since it will reduce churn in the git diff when adding new lines

**Comment by Lekensteyn** [comment]
LGTM, merge it whenever you are ready.

**Comment by bluca** [comment]
Thanks, tonight I'll do a test to make sure the 2 new commits are OK, push to the testing PPA and then merge if it's all right.

**Pull request merged**